src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -931,55 +965,11 @@ protected void UpdateState(Touch touchInfo = null)
         /// <since_tizen> 6 </since_tizen>
         private void Initialize()
         {
-            var style = (ButtonStyle)Style;
-
-            Extension = style.CreateExtension();
-
-            CreateComponents();
-
             EnableControlStatePropagation = true;
-
-            if (ButtonOverlayImage != null)
-            {
-                Add(ButtonOverlayImage);
-                ButtonOverlayImage.ApplyStyle(style.Overlay);
-            }
-
-            if (ButtonIcon != null)
-            {
-                Add(ButtonIcon);
-                ButtonIcon.ApplyStyle(style.Icon);
-                ButtonIcon.Relayout += OnIconRelayout;
```

Comment:
listening Icon's relayout signal is needed.
In the first, Button does not know the correct size of the Icon. (image is not loaded yet)
 it can look weird in the first if you remove  this line.

Please run [this](https://github.com/dalihub/nui-demo/blob/linux/wearable-samples/Button/ButtonWithOverlayAnimation/README.md) example and do clicking what the difference.